client/influxdb.go

```diff
@@ -180,6 +180,23 @@ func (c *Client) SetPrecision(precision string) {
 
 // Query sends a command to the server and returns the Response
 func (c *Client) Query(q Query) (*Response, error) {
+
```

Comment:
Can we remove these extraneous spaces? And instead of adding a new function `createHTTPRequest` and duplicating code, can you move the implementation of this function to `QueryContext` and have this method be:

```
func (c *Client) Query(q Query) (*Response, error) {
    return c.QueryContext(q, context.Background())
}
```

It seems like it would require less code and fewer changes.

Reply:
Done.